pymemcache/client/base.py

```diff
@@ -72,7 +72,7 @@ def _check_key(key, key_prefix=b''):
         except UnicodeEncodeError:
             raise MemcacheIllegalInputError("No ascii key: %r" % (key,))
     key = key_prefix + key
-    if b' ' in key:
+    if b' ' in key or b'\r\n' in key:
```

Comment:
After talking to folks in the memcached IRC channel, this is sufficient for security issues. But I just tested manually using both of them, and `\n` breaks the client for 1 call.. 

I will add a fix for this in this PR